### PR TITLE
roachtest: update version map and fixtures

### PR DESCRIPTION
**Pull request by alyshanjahani-crl**
This commit adds the recently release 20.2.8 to the version map.
And generates test fixtures.

Release note: None

### CODE REVIEW

**Review by pbardea — CHANGES_REQUESTED** [review]

**Comment by pbardea** [comment]
Also nit: the commit message says that it updates the 20.1 fixtures, when the 20.2 fixtures are being updated.

File: pkg/cmd/roachtest/versionupgrade.go [hunk 1]
```diff
@@ -90,11 +90,11 @@ func runVersionUpgrade(ctx context.Context, t *test, c *cluster, buildVersion ve
 	// the 20.2 release, this test will fail because it is missing a fixture for
 	// 20.1; run the test (on 20.1) with the bool flipped to create the fixture.
 	// Check it in (instructions will be logged below) and off we go.
-	if false {
+	if true {
```
[the remark this answers is not in the document]
Comment (alyshanjahani-crl) [reply pkg/cmd/roachtest/versionupgrade.go]: ah my bad, thx for catching that


> Also nit: the commit message says that it updates the 20.1 fixtures, when the 20.2 fixtures are being updated.

fixed, thx!

**Comment by alyshanjahani-crl** [comment]
Having trouble getting past a flaky test 
```
------- Stdout: -------
=== RUN   TestChangefeedMonitoring/sinkless
    changefeed_test.go:1697: expected > 0 got 0
    --- FAIL: TestChangefeedMonitoring/sinkless (10.83s)
```

**Comment by alyshanjahani-crl** [comment]
I'm getting a different test failure each time.
Is something up w/ master ? I've tried rebasing to get latest changes multiple times now.

@pbardea any suggestions?

**Review by pbardea — APPROVED** [review]
It looks like that changefeed test failure should be fixed by https://github.com/cockroachdb/cockroach/pull/64270. It looks like this PR isn't rebased on top of that change, giving it one more rebase off latest master should resolve that flake. You could also give it a chance at borsing since the failure is known.

Also as an aside, we typically push to a fork of the repo rather than create more branches in cockroachdb/cockroach and create a PR from the fork.

**Pull request merged**